js/src/setup-mc/setup-stepper/store-requirements/index.js

```diff
@@ -187,12 +102,15 @@ export default function StoreRequirements( { onContinue } ) {
 									setPhoneNumberReady( true )
 								}
 							/>
-							<PreLaunchChecklist />
+
 							<StepContentFooter>
 								<AppButton
 									isPrimary
 									loading={ adapter.isSubmitting }
-									disabled={ ! settingsSaved }
+									disabled={
```

Comment:
This might be a good idea, but I think it's unnecessary. Currently, the form can be submitted as long as the settings aren't being saved. Once saved, if there is some validation that fails, the `handleSubmitClick` callback will show validation errors rather than moving on to the next step. Therefore, I only think we need to disable this button if we're in the middle of recording some state that would change the validation (not something that is currently applicable).